brayovsky/pivotaly
# Enable copy story from backlog

#### What does this PR do? / Fixes issue
Adds a copy button to backlog stories that allows one to copy a story id
Solves issue #91 

#### How should this be manually tested?
Click on copy button on a story in the backlog view
The story id should be copied to the clipboard

#### Any background context you want to provide?
N/A

#### Screenshots (if appropriate)
<img width="645" alt="Screen Shot 2020-02-25 at 15 57 53" src="https://user-images.githubusercontent.com/19430730/75249609-9fe70200-57e7-11ea-89b1-d5fb1aa2537b.png">

#### Questions:
